core/src/main/resources/lib/hudson/newFromList/form.jelly

```diff
@@ -99,22 +97,5 @@ THE SOFTWARE.
       <input type="submit" name="Submit" value="${%OK}" id="ok" style="margin-left:5em" />
```

Comment:
**(not related to this PR)**

`style="margin-left:5em"`

I was not able to understand why this is done... I imagine it's a forgotten debug utility used during development. Once the makeButton is executed, the Ok button is moved by 5em to the left due to the removal of the margin-left. If you put a breakpoint before the `makeButton(...)` you will see the Ok with a space.
Does not really impact the end user, but it's fairly confusing for not apparent reasons. It was there since the beginning, in 2009 🕵️ Perhaps at the age of IE (50+% of market share), the display:none of the radio was problematic 🤷‍♂️